### ISSUE TITLE
Test-Application - HPC Module Maintenance/Update

### ISSUE THREAD

**Issue by thecyclingcyclopse** [brainhackorg/brainhack_cloud#79]
### HPC Module Maintenance / Update

This is a test during the maintenance/update of the HPC module. This module currently reflects computeCA exclusively and will be supplemented by information about the Brainhack Cloud. In order to document the procedure properly I need to apply the process a student would have to run and access to the Brainhack Cloud to test what parameters need changing, if any. Maybe you Brainhack Cloud admins can pitch in? In the best case the only thing that's different is the ssh connection?

### Is this a new project?

yes

### People involved and contact details (including email address!)

Thomas S. Hartmann
randominbox1@gmail.com 
@thecyclingcyclopse on GH

### Project Description

- What are you doing, for whom, and why?
Updating Brainhack HPC module for Brainhack School 2023 because it's fun!

### Link to project repository/sources

https://github.com/school-brainhack/school-brainhack.github.io/issues/167#issuecomment-1503375455

### What resources do you require?

HPC

### Anything else?

Actually I don't really require HPC, nor will the students require extensive compute power, but for the exercise it would be nice to have them access a cluster as intended by the exercise's creator.

edit:
minor typo

**Comment by stebo85** [comment]
Dear @thecyclingcyclopse,

I created a little teaching cluster for you. Can you send me your Public key - so I can add you to the admin account? Once this is done you can create user accounts for your students.

Kind regards
Steffen

**Comment by thecyclingcyclopse** [comment]
Hi Steffen,
thank you very much!
You can acquire my public key here:
https://drive.proton.me/urls/GMT93HV4TG#hR88ENXlYG9P
Kind regards,
Thomas

**Comment by stebo85** [comment]
Dear @thecyclingcyclopse,

I added your SSH key and you should now be able to connect to the admin account via:
ssh opc@140.238.144.20

Can you let me know if this works?

Thank you
Steffen

**Comment by stebo85** [comment]
@thecyclingcyclopse,

Once the login works for you, you can create normal users with the following command:
```
cluster user add name
```

These users can be created with a password only - so they don't need an SSH key (I enabled this just now)

Let me know if you need anything else

**Comment by thecyclingcyclopse** [comment]
Hi Steffen! Connection and user adding works fine! Thank you very much!

**Comment by stebo85** [comment]
This project is now closed and all resources on the Oracle Cloud will be deleted within the next few days.
